samples/web/content/apprtc/index.html

```diff
@@ -21,7 +21,19 @@
 </head>
 
 <body>
-
+  <script>
+    // Keep this in sync with the HTML element id attributes.
+    var uiConstants = {
```

Comment:
Chrome app's can't have inline scripts, so this may need to be moved to a js file if we want to keep the html the same for both.